src/gradient/gradient.class.ts

```diff
@@ -1,493 +1,342 @@
-//@ts-nocheck
 
-import { Color } from "../color";
 
+import { fabric } from "../../HEADER";
+import { Color } from "../color";
+import { iMatrix } from "../constants";
+import { parseTransformAttribute } from "../parser/parseTransformAttribute";
+import { matrixToSVG, populateWithProperties } from "../util";
```

Comment:
please import thos from the respective files, wherever are them now, if are not exported yet keep using fabric.util...

Reply:
What's wrong?

Reply:
somehow it does create issues when rollup build, mostly because point is referenced inside of some util and so when transposing some utils to TS, the import order change and the build break

On top of that i don't know if someone using the TS file directly will have tree shaking available, and i don't want them to import the whole util folder if they are trying to import only the utils they need